src/ts/components/errormessageoverlay.ts

```diff
@@ -99,22 +101,25 @@ export class ErrorMessageOverlay extends Container<ErrorMessageOverlayConfig> {
     let config = <ErrorMessageOverlayConfig>this.getConfig();
 
     player.on(player.exports.Event.Error, (event: ErrorEvent) => {
-      let message = event.message;
+      let description = ErrorEventUtil.defaultErrorMessagesMapping[event.code] as string;
 
-      // Process message translations
+      let name = event.name;
+      let message = description + '\n(' + name + ')'; // default error message style
+
+      // Custom error message handling
       if (config.messages) {
         if (typeof config.messages === 'function') {
-          // Translation function for all errors
-          message = config.messages(event);
+          // Custom function for all errors
```

Comment:
Why did you change the wording of the comments around here? The old ones are still more fitting.

Reply:
translation was confusing me because the meaning off translating to a different language.

Reply:
"Translation" doesn't have to refer to different spoken languages. You can also translate an error code/event to an error message.

Reply:
Also the basic idea was indeed to allow translations into different languages.